docs/tutorials/FlowTutorial.md

```diff
@@ -0,0 +1,1630 @@
+|--------------------------|----------------------------------------------------------|
+| `5_route_drc.rpt`        | DRC violations if occurred                               |
+| `final_clocks.webp`      | OR extracted image reference after clock tree synthesis. |
+| `final_resizer.webp`     | OR extracted image reference after resizer.              |
+| `synth_check.txt`        | Synthesis warning/error messages                         |
+| `antenna.log`            | Antenna check log report                                 |
+| `final_placement.webp`   | Extracted image after final placement                    |
+| `final.webp`             | Extracted image after routing.                           |
+| `synth_stat.txt`         | Post synthesis design statistics log saved here.         |
+
+The flow completes with the message below by creating a merged final GDS file.
+
+```
+[INFO] Writing out GDS/OAS
+'results/sky130hd/ibex/base/6_1_merged.gds'
+cp results/sky130hd/ibex/base/6_1_merged.gds
+results/sky130hd/ibex/base/6_final.gds
+```
+
+## Viewing Results And Logs
+
+The ORFS flow appends a suffix to each flow stage as shown below to
+indicate the position in the RTL-GDS flow. This makes it easier to
+understand and debug each flow stage in case of failure.
+
+View `ibex` design logs.
+
+```
+ls logs/sky130hd/ibex/base/
+```
+
+The log structure as follows:
+
+```
+1_1_yosys.log
+2_1_floorplan.log 2_1_floorplan.json 2_2_floorplan_io.log
+2_2_floorplan_io.json 2_3_tdms_place.log 2_3_tdms.json
+2_4_mplace.log 2_4_mplace.json
+2_5_tapcell.log 2_5_tapcell.json 2_6_pdn.log 2_6_pdn.json
+3_1_place_gp.log 3_1_place_gp.json 3_2_place_iop.log
+3_2_place_iop.json 3_3_resizer.log 3_3_resizer.json 3_4_opendp.log
+3_4_opendp.json
+4_1_cts.log 4_1_cts.json 4_2_cts_fillcell.log 4_2_cts_fillcell.json
+5_1_fastroute.log 5_1_fastroute.json 5_2_TritonRoute.log
+5_2_TritonRoute.json 6_1_merge.log 6_report.log 6_report.json
+```
+
+### Area
+
+View design area and its core utilization.
+
+```
+make gui_final
+report_design_area
+```
+
+View the resulting area as:
+
+```
+Design area 191262 u^2 30% utilization.
+```
+
+### Timing
+
+User can view flow results using the command interface from the shell or
+the OpenROAD GUI to further visualize, and debug. Learn more about the
+[GUI](../../src/gui/README.md)
+
+```
+make gui_final
+```
+
+Type following commands in `Tcl Commands` section of GUI:
+
+```
+report_worst_slack
+report_tns
+report_wns
+```
+
+Note the worst slack, total negative slack and worst negative slack:
+
+```
+worst slack -1.35
+tns -95.25
+wns -1.35
+```
+
+To learn more about visualizing and tracing time paths across the design
+hierarchy refer to the OpenROAD [GUI](../../src/gui/README.md)
+
+### Power
+
+Now use the report command to view individual power components i.e.
+sequential, combinational, macro and power consumed by I/O pads.
+
+```
+report_power
+```
+
+The power output as follows:
+
+```
+finish report_power

```

Comment:
This isn't part of the output

Reply:
@arlpetergadfort  this is I captured from ibex design 6_report.log.

Reply:
@vijayank88 Peter is right, the `finish report_power` is part of the `final_report.tcl` not the `report_power` command.

Reply:
report_power comes from final_metrics.tcl
I don't understand what should be updated here @vvbandeira

Reply:
@vvbandeira  understood, finish report power should not be part of log here